config/v1/zz_generated.crd-manifests/0000_10_config-operator_01_infrastructures-CustomNoUpgrade.crd.yaml

```diff
@@ -784,7 +784,7 @@ spec:
                                     /<datacenter>/network/<portgroup>.
                                   items:
                                     type: string
-                                  maxItems: 10
+                                  maxItems: 1
```

Comment:
i'm unsure why this snuck in

Reply:
Please make sure you're rebased on the latest tip of master and try the generation again 🤔